lib/p2p/Pool.ts

```diff
@@ -528,9 +542,12 @@ class Pool extends EventEmitter {
       this.logger.verbose(`opened connection to ${peer.nodePubKey} at ${addressUtils.toString(peer.address)}`);
       this.peers.add(peer);
 
-      // request peer's orders and known nodes
+      // request peer's orders
       peer.sendPacket(new packets.GetOrdersPacket({ pairIds: this.handshakeData.pairs }));
-      peer.sendPacket(new packets.GetNodesPacket());
+      if (this.config.discover) {
+        // request peer's known nodes only if p2p.discover option is true
```

Comment:
comment mention p2p.discover while if is using this.config.discover

Reply:
I couldn't get what you mean exactly @offerm  ?

Reply:
this.config has a comment which tells that;
>   /** Points to config comes during construction. */
And in constructor it's a `PoolConfig`

Reply:
Ignore. I'm good.